ViAn/Analysis/analysismethod.h

```diff
@@ -35,7 +35,8 @@ class AnalysisMethod : public QObject ,public QRunnable{
     bool paused = false;
 
 protected:
-    AnalysisInterval interval;
+    std::pair<int, int> interval;

```

Comment:
Why not use the AnalysisInterval? It contains the same information

Reply:
I felt like i didn't need/want the extra things in the AnalysisInterval. The interval also isn't an AnalysisInterval in the way we used it before. And lastly, before the interval was sometimes a pair and sometimes an AnalysisInterval which wasn't really necessary. Now it's the same type all the time.